app/helpers/container_summary_helper.rb

```diff
@@ -71,6 +71,19 @@ def textual_container_image_registries
     textual_link(@record.container_image_registries)
   end
 
+  def textual_tags
+    label = "#{session[:customer_name]} Tags"
+    h = {:label => label}
+    tags = session[:assigned_filters]
+    if tags.empty?
+      h[:image] = "smarttag"
+      h[:value] = "No #{label} have been assigned"
+    else
+      h[:value] = tags.sort_by { |category, assigned| category.downcase }.collect { |category, assigned| {:image => "smarttag", :label => category, :value => assigned } }
+    end
+    h
+  end
```

Comment:
This seems to be the standard way to get tags for an entity however tags wont show for entities other than ```ems_container```. What are the prerequisites for showing them? 
Do I need to set ```session[:assigned_filters]``` in some particular way?